FuzzySetLib/Distances.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FuzzySetLib
+{
+    public class Distances
+    {
+        public static float[,] HammingDistance(Func<float, float, float> norm, Func<float, float, float> impl, float[,] Q, float[,] R)
+        {
+            float[,] distances = new float[Q.GetLength(0), R.GetLength(0)];
+
+            var objectBounds = Approximations.ObjectBoundApproximation(norm, impl, Q, R);
+            var propertyBounds = Approximations.PropertyBoundApproximation(norm, impl, Q, R);
+
+            for(int i=0;i<distances.GetLength(0);i++)
+            {
+                for(int j=0;j<distances.GetLength(1);j++)
+                {
+                    for(int k=0;k<objectBounds.GetLength(1);k++)
```

Comment:
nit: spacje, też w innych miejscach
```suggestion
                    for(int k = 0 ; k < objectBounds.GetLength(1); k++)
```